lib/WasmReader/WasmLimits.h

```diff
@@ -16,7 +16,7 @@ namespace Wasm {
         static const uint32 MaxGlobals = 1000000;
         static const uint32 MaxDataSegments = 100000;
         static const uint32 MaxElementSegments = 10000000;
-        static const uint32 MaxTableSize = 10000000;
+        static const uint32 MaxTableSize = DEFAULT_CONFIG_WasmMaxTableSize;
```

Comment:
This is now unused, but I see why you kept it. Going forward, is this maybe something we want to change in how ConfigFlags list can work, such that we could privately scope these things somewhere and have the config flag list include them somehow?